scripts/fetch-asyncapi-example.js

```diff
@@ -24,12 +24,20 @@ const TEMP_ZIP_NAME = 'spec-examples.zip';
 
 const fetchAsyncAPIExamplesFromExternalURL = () => {
   return new Promise((resolve, reject) => {
-    request(SPEC_EXAMPLES_ZIP_URL)
-      .pipe(fs.createWriteStream(TEMP_ZIP_NAME))
-      .on('close', () => {
-        console.log('Fetched ZIP file');
-        resolve();
-      }).on('error', reject);
+    fetch(SPEC_EXAMPLES_ZIP_URL)
```

Comment:
Error handling can be improved by wrapping by a try/catch block (see: https://github.com/node-fetch/node-fetch?tab=readme-ov-file#handling-exceptions)